compute/ARMComputeEx/src/runtime/CL/functions/CLSplitVEx.cpp

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Would you add License like the other files? e.g. CLGatherEx.cpp

Reply:
Yes, I will add the license as soon as I have passed the CI tests. Thanks for spotting.

Reply:
@intom, PTAL